src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -86,49 +87,56 @@ public void LogFunctionExecutionEvent(string executionId, string siteName, int c
                     WriteEvent(57909, executionId, siteName, concurrency, functionName, invocationId, executionStage, executionTimeSpan, success);
                 }
             }
+        }
+
+        [EventSource(Guid = "a7044dd6-c8ef-4980-858c-942d972b6250")]
+        [SuppressMessage("Microsoft.StyleCop.CSharp.NamingRules", "SA1306:FieldNamesMustBeginWithLowerCaseLetter", Justification = "Need to use Pascal Case for MDS column names")]
+        public class FunctionsSystemLogsEventSource : EventSource
+        {
+            internal static readonly FunctionsSystemLogsEventSource Instance = new FunctionsSystemLogsEventSource();
 
-            [Event(65520, Level = EventLevel.Verbose, Channel = EventChannel.Operational)]
-            public void RaiseFunctionsEventVerbose(string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string hostVersion, string eventTimestamp)
+            [Event(65520, Level = EventLevel.Verbose, Channel = EventChannel.Operational, Version = 1)]
+            public void RaiseFunctionsEventVerbose(string SubscriptionId, string AppName, string FunctionName, string EventName, string Source, string Details, string Summary, string HostVersion, string EventTimestamp)
             {
                 if (IsEnabled())
                 {
-                    WriteEvent(65520, subscriptionId, appName, functionName, eventName, source, details, summary, hostVersion, eventTimestamp);
+                    WriteEvent(65520, SubscriptionId, AppName, FunctionName, EventName, Source, Details, Summary, HostVersion, EventTimestamp);
                 }
             }
 
-            [Event(65521, Level = EventLevel.Informational, Channel = EventChannel.Operational)]
-            public void RaiseFunctionsEventInfo(string subscriptionId, string appName, string functionName, string eventName, string source, string details, string summary, string hostVersion, string eventTimestamp)
+            [Event(65521, Level = EventLevel.Informational, Channel = EventChannel.Operational, Version = 1)]
```

Comment:
Yes. This is a feature in MDS. Any time we add new columns version needs to be updated.